app/assets/stylesheets/admin.scss

```diff
@@ -96,3 +96,29 @@
 .admin-navigation-links {
   width: 240px;
 }
+
+.config-authentication__row {
+  &--select {
+    display: grid;
+    grid-template-columns: auto 1fr auto;
+    grid-gap: var(--su-3);
+    padding: var(--su-4);
+    align-items: center;
+  }
+
+  &--title {
+    font-weight: var(--fw-medium);
+  }
+
+  &--subtitle {
+    font-size: var(--fs-s);
+    color: var(--base-60);
+  }
```

Comment:
These two elements feel like won't be used anywhere else, so maybe instead of writing CSS here you could apply appropriate utility classes in the html?

```
<element class="fw-medium">
<element class="fs-s color-base-60">
```

I tend to use utility classes over building own separate classes, especially when it's for very specific thing (your class name `config-authentication` suggest it will be used only here), when I have only one or two styles to apply.

Reply:
I can update these to utility classes.

Tangent, but I think we could develop some of these typographic styles into Crayons components. For example, a row is a pretty typical UI component and it will be sensible to turn this into a repeatable pattern that we use in multiple places and can reference easily.